tests/s-thread-pthread_exit.c

```diff
@@ -0,0 +1,27 @@
+#include <pthread.h>
+
+#define NUM_THREAD 4
```

Comment:
Hmm.. do you really need that number of threads?  Isn't it reproducible if it creates a single thread?